src/topology/sheaves/skyscraper.lean

```diff
@@ -64,6 +64,51 @@ point, then the skyscraper presheaf `𝓕` with value `A` is defined by `U ↦ A
     { rw [dif_neg hW], apply ((if_neg hW).symm.rec terminal_is_terminal).hom_ext }
   end }
 
+section
+
+variables [Π (U : opens (Top.of (punit : Type u))), decidable (punit.star ∈ U)]
+
+def skyscraper_presheaf_iso_pullback :
+  skyscraper_presheaf p₀ A ≅
+  { to_fun := λ _, p₀, continuous_to_fun := by continuity } _*
+    (skyscraper_presheaf punit.star A : presheaf C (Top.of (punit : Type u))) :=
```

Comment:
I think you can show (and it's quicker to show) they are actually equal, maybe even defeq??

Reply:
and it's pushforward, not pullback.

Reply:
Turns out they're defeq apart from the choice of decidable instance:
```
def skyscraper_presheaf_iso_pullback :
  skyscraper_presheaf p₀ A ≅
  continuous_map.const (Top.of _) p₀ _* (skyscraper_presheaf punit.star A) :=
eq_to_iso begin
  convert_to @skyscraper_presheaf X p₀ (λ _, classical.dec _) C _ A _ =
    continuous_map.const _ p₀ _*
    (@skyscraper_presheaf (Top.of _) punit.star (λ _, classical.dec _) C _ A _),
  { congr }, { congr }, { refl },
end
```
It's a bit strange and inconvenient that `simp_rw [this]` can't simplify when I have 
```
  have : ∀ p (h : decidable p), h = classical.dec _ := λ _ _, subsingleton.elim _ _,
```
in the context.

Reply:
I think using classical instance is not a good idea, because most files in mathlib doesn't use classical instance, so there is a risk of Lean complaining synthesised instance and inferred instance not being the same. (I don't actually know what I am talking about)